### DISCUSSION TITLE
Data Caching Webgl

### DISCUSSION THREAD

**Discussion by Mir4nda** [jeffreylanters/react-unity-webgl#436] [Question and Answer]
Is it possible to run a build with data caching enabled before building on Unity?

I try to load my build on my IIS machine and ever refresh the page, takes 30s around to load, I mean, it is ignoring the data caching option I think.

Could you help me please?

Thank you

**Comment by jeffreylanters** [comment]
Hi! This should be configured on the server. Oh what kind of server are you hosting your web application?

**Reply by Mir4nda** [reply]
This is a IIS on Windows Server, is it possible to use, do you know? :/

**Comment by Mir4nda** [comment]
Hello,

I apologize for insisting on my problem.

I've been testing and I've noticed that:

-if I put the webgl export with the index.html that unity generates I can work with the cache, that is, the ".wasm" and ".data" files are stored on the client side. Even if you F5 the page, it will no longer download from the server.
image: https://i.imgur.com/PegwE2r.png

-if you use your unityContext in react, the ".data" file is always downloaded, whenever you do F5 to the page... and it wasn't supposed to.
Image: https://i.imgur.com/mLXCSRV.png

Can you understand what could be influencing this behavior?
I reinforce that I am using IIS from windows server to host the project and I used the default configuration that unity recommends:
https://docs.unity3d.com/Manual/webgl-server-configuration-code-samples.html

Thanks for your attention 
Best Regards

**Reply by 0xNunoMiranda** [reply]
hello :) Do you have some update about this? Thanks for your attention

**Comment by programmer119** [comment]
have no another idea?

**Comment by ibrahim-qordhawi** [comment]
Try adding company name and product name in the unity config like this #388. This worked for me.
